stable/telegraf/values.yaml

```diff
@@ -0,0 +1,491 @@
+      metric_batch_size: 1000
+      metric_buffer_limit: 10000
+      collection_jitter: "0s"
+      flush_interval: "10s"
+      flush_jitter: "0s"
+      precision: ""
+      debug: false
+      quiet: false
+      logfile: ""
+      hostname: ""
+      omit_hostname: false
+    outputs:
+      influxdb:
+        urls:
+          - "http://influxdb-influxdb.tick:8086"
+        database: "telegraf"
+##        retention_policy: ""
+##        write_consistency: "any"
+##        timeout: "5s"
+##        username: "telegraf"
+##        password: "metricsmetricsmetricsmetrics"
+##        user_agent: "telegraf"
+##        udp_payload: 512
+##        ssl_ca: "/etc/telegraf/ca.pem"
+##        ssl_cert: "/etc/telegraf/cert.pem"
+##        ssl_key: "/etc/telegraf/key.pem"
+##        insecure_skip_verify: false
+##      amqp:
+##        url: "amqp://localhost:5672/influxdb"
+##        exchange: "telegraf"
+##        auth_method: "PLAIN"
+##        routing_tag: "host"
+##        retention_policy: "default"
+##        database: "telegraf"
+##        precision: "s"
+##        ssl_ca: "/etc/telegraf/ca.pem"
+##        ssl_cert: "/etc/telegraf/cert.pem"
+##        ssl_key: "/etc/telegraf/key.pem"
+##        insecure_skip_verify: false
+##        data_format: "influx"
+##      kafka:
+##        brokers: 
+##          - "localhost:9092"
+##        topic: "telegraf"
+##        routing_tag: "host"
+##        compression_codec: 0
+##        required_acks: -1
+##        max_retry: 3
+##        ssl_ca: "/etc/telegraf/ca.pem"
+##        ssl_cert: "/etc/telegraf/cert.pem"
+##        ssl_key: "/etc/telegraf/key.pem"
+##        insecure_skip_verify: false
+##        data_format: "influx"
+##      mqtt:
+##        servers: 
+##          - "localhost:1883"
+##        topic_prefix: "telegraf"
+##        username: "telegraf"
+##        password: "metricsmetricsmetricsmetrics"
+##        ssl_ca: "/etc/telegraf/ca.pem"
+##        ssl_cert: "/etc/telegraf/cert.pem"
+##        ssl_key: "/etc/telegraf/key.pem"
+##        insecure_skip_verify: false
+##        data_format: "influx"
+##      nats:
+##        servers: 
+##          - "nats://localhost:4222"
+##        username: ""
+##        password: ""
+##        subject: "telegraf"
+##        ssl_ca: "/etc/telegraf/ca.pem"
+##        ssl_cert: "/etc/telegraf/cert.pem"
+##        ssl_key: "/etc/telegraf/key.pem"
+##        insecure_skip_verify: false
+##        data_format: "influx"
+##      nsq:
+##        server: "localhost:4150"
+##        topic: "telegraf"
+##        data_format: "influx"
+    inputs:
+      cpu:
+        percpu: true
+        totalcpu: true
+        collect_cpu_time: false
+      disk:
+        ignore_fs: 
+          - "tmpfs" 
+          - "devtmpfs"
+      diskio:
+      docker:
+        endpoint: "unix:///var/run/docker.sock"
+        container_names: []
+      kernel:
+      mem:
+      processes:
+      swap:
+      system:
+      kubernetes:
+        url: "http://$NODE_IP:10255"
```

Comment:
You can reference `ENV` in the telegraf configuration file. That would get evaluated when telegraf starts up. I haven't had an issue with this approach before. Are you seeing:
```
2016-12-05T20:56:40Z E! ERROR in input [inputs.kubernetes]: Errors encountered: [error making HTTP request to http://$NODE_IP:10255/stats/summary: dial tcp [::1]:10255: getsockopt: connection refused]```